server/src/main/resources/db/V3__create_pulse_response_table.sql

```diff
@@ -0,0 +1,10 @@
+drop table if exists pulse_response;
+CREATE TABLE pulse_response (
+   id varchar PRIMARY KEY,
+   submissionDate date,
+   updatedDate date,
+   teamMemberId varchar REFERENCES member_profile(uuid),
+   questionResponse varchar,
```

Comment:
Yank this column as well...and you're going to need to make this file V4, since Holly's PR went in first.